cmake/TestImpactFramework/ConsoleFrontendConfig.in

```diff
@@ -103,7 +105,9 @@
         "root": "${python_temp_dir}",
         "run_artifact_dir": "${LY_TEST_IMPACT_PYTHON_TEST_RUN_DIR}",
         "coverage_artifact_dir": "${python_temp_dir}/Coverage",
-        "enumeration_cache_dir": "${python_temp_dir}/EnumerationCache"
+        "enumeration_cache_dir": "${python_temp_dir}/EnumerationCache",
+        "change_list" : "${python_temp_dir}/Changelists",
```

Comment:
nit: This is super minor but if we're using `change_list` here to separate the words then ideally we should have `ChangeLists` in the CamelCase version (or lowercase should be `changelists` - I don't really mind which but it's good to be consistent when dealing with word boundaries